v19.2/vectorized-execution.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Vectorized Query Execution
+summary: The CockroachDB vectorized SQL query execution engine processes query plans using a column-oriented model to improve performance.
+toc: true
+---
+
+ <span class="version-tag">New in v19.2:</span> CockroachDB supports column-oriented ("vectorized") query execution on all [supported SQL operations](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html).
+
+ Many SQL databases, including MySQL, PostgreSQL, and all versions of CockroachDB released prior to 19.2, execute [query plans](https://en.wikipedia.org/wiki/Query_plan) one row of table data at a time. The CockroachDB vectorized execution engine dramatically improves performance over row-oriented execution by processing each component of a query plan on type-specific batches of column data. Column-oriented execution models contrast with row-oriented execution models, which can offer good performance for [online transaction processing (OLTP)](https://en.wikipedia.org/wiki/Online_transaction_processing) queries, but suboptimal performance for [online analytical processing (OLAP)](https://en.wikipedia.org/wiki/Online_analytical_processing) queries.
```

Comment:
The opening sentence would not lose any important information, and would be easier to read as, "Many SQL databases execute query plans one row at a time"
I think we should link use of "column-oriented execution models" to something like https://en.wikipedia.org/wiki/Column-oriented_DBMS#Column-oriented_systems, and use of "row-oriented" to something like https://en.wikipedia.org/wiki/Column-oriented_DBMS#Row-oriented_systems.
I think the multi-clause sentence "Column-oriented execution models contrast with ..." is kinda hard to follow the logic of.  It should be broken into shorter, more declarative sentences. E.g., "Row-oriented execution models can offer good performance on [OLTP]() queries, but may not perform well on [OLAP]() queries.  Therefore we've added support for columnar ("vectorized" execution, which dramatically improves performance for OLAP ..."